src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -107,7 +107,7 @@ public async Task<bool> PushMetadataAsync(CancellationToken cancellationToken)
                 var buildId = await GetBuildId(dep, client, cancellationToken);
                 if (buildId == null)
                 {
-                    Log.LogWarning($"Asset '{dep.Name}@{dep.Version}' not found in BAR, ignoring.");
+                    Log.LogMessage($"Asset '{dep.Name}@{dep.Version}' not found in BAR, most likely this is an external dependency, ignoring...");
```

Comment:
This message will only be shown if verbose is normal or diag. Just because this is relevant, should it have been important?

Reply:
Yea, this should be a high importance message.

Reply:
https://github.com/dotnet/arcade-services/pull/283